pegasus/sites.v3/code.org/public/css/container-responsive-full-width.css

```diff
@@ -0,0 +1,26 @@
+/* Allows for full-width banners and sections on Pegasus pages */
```

Comment:
can you say more in the comments about how this is different from container-responsive -- I see that this max width of 970px does not apply: https://github.com/code-dot-org/code-dot-org/blob/b90a9daa5c1dd102782e8a811989f8529e620af7/pegasus/sites.v3/code.org/public/css/brstrap.css#L79 are there other intended differences?

Reply:
Yes, definitely! Before I commit, does this make more sense:

```
/* 
  Allows for full-width banners and sections on Pegasus pages.
  
  Should be used on pages that:
  - have a full-width hero banner
  - use a background color to delineate between sections

  This differs from the .container_responsive class on brstrap.css
  by making the container the full-width of the screen at any size.

  The container_responsive class uses a defined width that forces margins
  on each side which makes a full-width design impossible.

  The "wrapper" class on a section element now acts as the container,
  allowing each section to be full-width.
*/
```

Reply:
💯  looks great!

Reply:
Agreed, very helpful comment! Thank you

Reply:
Just added 🙌